### DISCUSSION TITLE
Support Regional Settings Date&Time formats

### DISCUSSION THREAD

**Discussion by zockinger** [tnodir/fort#147] [Q&A] [answered]
Various date/time fields are always displayed in US format 
Examples are the "Creation Time" field in the Programs Window  and "Time" in the Blocked Connections Tab
Displaying those fields using the user's Date/Time format would be preferable

**Comment by tnodir** [comment, marked as answer]
Please use Options: Global: "Use System Regional Settings" flag:

![optins-system-region](https://github.com/tnodir/fort/assets/303655/3e5ba11d-106c-4557-b389-8378e7e5c92d)

**Reply by zockinger** [reply]
Thanks for the quick reply and sorry for wasting your time
